libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -125,27 +125,26 @@ public static String getJourneyParameter(URI journeyUri, String key) {
 
     public static String getScoreType(ProcessRequest request)
             throws HttpResponseExceptionWithErrorBody {
-        return extractValueFromLambdaInput(
-                request, "scoreType", ErrorResponse.MISSING_SCORE_TYPE, String.class);
+        return extractValueFromLambdaInput(request, "scoreType", ErrorResponse.MISSING_SCORE_TYPE);
     }
 
     public static Integer getScoreThreshold(ProcessRequest request)
             throws HttpResponseExceptionWithErrorBody {
         return extractValueFromLambdaInput(
-                request, "scoreThreshold", ErrorResponse.MISSING_SCORE_THRESHOLD, Integer.class);
+                request, "scoreThreshold", ErrorResponse.MISSING_SCORE_THRESHOLD);
     }
 
-    public static Boolean getIsUserInitiated(ProcessRequest request)
+    public static boolean getIsUserInitiated(ProcessRequest request)
             throws HttpResponseExceptionWithErrorBody {
-        return extractValueFromLambdaInput(
-                request,
-                "isUserInitiated",
-                ErrorResponse.MISSING_IS_USER_INITIATED_PARAMETER,
-                Boolean.class);
+        return Boolean.TRUE.equals(
+                extractValueFromLambdaInput(
+                        request,
+                        "isUserInitiated",
+                        ErrorResponse.MISSING_IS_USER_INITIATED_PARAMETER));
     }
 
     private static <T> T extractValueFromLambdaInput(
-            ProcessRequest request, String key, ErrorResponse errorResponse, Class<T> returnType)
```

Comment:
Presumably this was intended to check that the value actually had the right type, but now we just don't bother? - Relying on a runtime exception in the (unlikely) case that it has the wrong type

Reply:
It was included to set the value for <T>. I think it was me failing to realised that <T> would be set by the return type defined when calling the method. This change is how I should have done it the first time.